setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup
+
+setup(
+    name="noobaa-sa-ci",
+    version="0.1",
+    packages=[""],
+    url="",
+    license="MIT",
+    author="Noobaa SA QE",
+    author_email="ocs-ci@redhat.com",
+    description=(
+        "Noobaa Standalone(SA) infra is used to deploy, upgrade and destroy NooBaa standalone on "
+        "RHEL/CentOS stream platforms"
+    ),
+    install_requires=[
+        "mergedeep",
+        "pytest",
+        "pynpm",
+        "pyyaml"
```

Comment:
Do we want to add requirements for common-ci-utils here?

Reply:
done